src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV6.sql

```diff
@@ -267,6 +263,12 @@ BEGIN
         AND CF.StudyInstanceUid = DI.StudyInstanceUid
         AND CF.SeriesInstanceUid = DI.SeriesInstanceUid
         AND CF.SopInstanceUid = DI.SopInstanceUid
+    
+    -- Delete FileProperties of instance
```

Comment:
Can we move this above the `ChangeFeed` operations as I still want to minimize the contention